File: Unidad_3/C#/Programa18_U3.cs
using System;

namespace Programa18_U3
{
    class Program
    {
        static void Main(string[] args)
        {
            double salarioInicial = 1500;
            double tasaIncremento = 0.10;
            int numAños = 6;

            Console.WriteLine("Salario inicial del profesor: $" + salarioInicial);

            for (int i = 1; i <= numAños; i++)
            {
                salarioInicial = salarioInicial + (salarioInicial * tasaIncremento);
                Console.WriteLine("Salario en el año " + i + ": $" + salarioInicial);
            }

            Console.WriteLine("Salario después de " + numAños + " años: $" + salarioInicial);
        }
    }
}

File: Unidad_3/Java/Programa18_U3.java

package com.mycompany.programa18_u3;


public class Programa18_U3 {

   
    public static void main(String[] args) {
        double salarioInicial = 1500; 
        double tasaIncremento = 0.10; 
        int numAños = 6; 
        
        System.out.println("Salario inicial del profesor: $" + salarioInicial);

        for (int i = 1; i <= numAños; i++) {
            salarioInicial = salarioInicial + (salarioInicial * tasaIncremento);
            System.out.println("Salario en el año " + i + ": $" + salarioInicial);
        }

        System.out.println("Salario después de " + numAños + " años: $" + salarioInicial);
    }
}
